content/en/remote-network-agent/overview.md

```diff
@@ -1,35 +1,45 @@
 ---
-title: Remote Network Agent Installation Overview
+title: Remote Network Agent Overview
 linktitle: Overview
 weight: 1
 categories: ["Remote Network Agent", "Features", "Concepts"]
 tags: ["Remote Network Agent"]
 description: >
-  Learn about the various methods to install an Armory CD-as-a-Service Remote Network Agent in your Kubernetes cluster: kubectl, Helm, Armory CLI, or a UI wizard.
+  a logicless network relay that enables Armory CD-as-a-Service to integrate with privately networked resources such as Jenkins, Prometheus, Kubernetes clusters, etc.
 ---
 
-## Remote Network Agent installation methods
+## Overview
 
-By default, you install a Remote Network Agent (RNA) with full access to your cluster. At a minimum, the RNA needs permissions to create, edit, and delete all `kind` objects that you plan to deploy with CD-as-a-Service, in all namespaces you plan to deploy to. The RNA also requires network access to any monitoring solutions or webhook APIs that you plan to forward through it.
+Armory's Remote Network Agent (RNA) is a simple, lightweight, and reliable agent that resides within your privately networked Kubernetes cluster. It facilitates the orchestration of deployments by providing CD-as-a-Service. RNA enables seamless communication with resources within your private network, empowering CD-as-a-Service to enable use cases such as executing Prometheus queries or initiating Jenkins jobs from within the cluster. RNA supports both `amd64` and `arm64` based architectures, ensuring compatibility across a range of systems.
 
-You can install the Remote Network Agent (RNA) in your Kubernetes cluster using one of the following:
+The RNA is installed with a logical name, allowing for easy identification of a cluster. This name is then used as an `account` when defining `targets` in your deployment configuration. Armory recommends installing a single agent per Kubernetes cluster to maintain simplicity and consistency.
```

Comment:
how does it correlate with number of replicas within the cluster? do you mean by "single agent per cluster" a single deployment with two replicas or did we change the number of replicas?

Reply:
![image](https://github.com/armory/docs-cdaas/assets/711726/1fab217c-f04d-4d8e-ae1b-56a93dafe2ec)